paasta_tools/cli/cmds/status.py

```diff
@@ -1796,24 +1796,35 @@ def print_cassandra_status(
     for node in nodes:
         ip = node.get("ip")
         err = node.get("error")
+        startAge = "None"
```

Comment:
nit: we tend to use snake_case for variable names

Reply:
Good catch. I converted them and pushed the change as a new commit.